LocalNotifications/LocalNotifications/LocalNotifications.Android/MainActivity.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Android.App;
+using Android.Content;
+using Android.Content.PM;
+using Android.OS;
+using Xamarin.Forms;
+
+namespace LocalNotifications.Droid
+{
+    [Activity(Label = "LocalNotifications",
+        Icon = "@mipmap/icon",
+        Theme = "@style/MainTheme",
+        MainLauncher = true,
+        LaunchMode = LaunchMode.SingleTop,
+        ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation)]
+    public class MainActivity : global::Xamarin.Forms.Platform.Android.FormsAppCompatActivity
+    {
+        protected override void OnCreate(Bundle savedInstanceState)
+        {
+            TabLayoutResource = Resource.Layout.Tabbar;
+            ToolbarResource = Resource.Layout.Toolbar;
+
+            base.OnCreate(savedInstanceState);
+
+            global::Xamarin.Forms.Forms.Init(this, savedInstanceState);
+            LoadApplication(new App());
+
+            CreateNotificationFromIntent(Intent);
+        }
+
+        protected override void OnNewIntent(Intent intent)
+        {
+            CreateNotificationFromIntent(intent);
+        }
+
+        void CreateNotificationFromIntent(Intent intent)
+        {
+            if(intent?.Extras != null)
+            {
+                var title = intent.Extras.GetString(AndroidNotificationManager.TitleKey);
```

Comment:
We've had complaints, from time to time, about our use of vars not making it clear what the returned type is. So probably best to make this a string.